tmt/tests/booted/test-20-image-pushpull-upgrade/images/01-bootc-derived/usr/lib/bootc/kargs.d/05-testkargs.toml

```diff
@@ -0,0 +1 @@
+kargs = ["testarg=foo", "othertestkarg", "thirdkarg=bar"]
```

Comment:
Not related to this specific line but just a note: since this PR is changing *all* of the tests it's going to be an ongoing merge conflict fest.

One thing that would likely here is to split up any "prep" changes that can land independently first